internal/store/bundle.go

```diff
@@ -111,15 +112,30 @@ func (b *bundleStore) List() ([]reference.Reference, error) {
 }
 
 // Remove removes a bundle from the bundle store.
-func (b *bundleStore) Remove(ref reference.Reference) error {
+func (b *bundleStore) Remove(ref reference.Reference, force bool) error {
 	if id, ok := ref.(ID); ok {
-		if len(b.refsMap[id]) == 0 {
+		refs := b.refsMap[id]
+		if len(refs) == 0 {
 			return fmt.Errorf("no such image %q", reference.FamiliarString(ref))
-		} else if len(b.refsMap[id]) > 1 {
+		} else if len(refs) > 1 {
+			var failure error
+			if force {
+				toDelete := append([]reference.Reference{}, refs...)
+				for _, r := range toDelete {
+					if err := b.doRemove(r); err != nil {
+						failure = err
```

Comment:
Here you will drop any other errors and only return the last failure.
You can use https://github.com/hashicorp/go-multierror to keep them:

```go
var failures *multierror.Error
toDelete := append([]reference.Reference{}, refs...)
for _, r := range toDelete {
        if err := b.doRemove(r); err != nil {
                failures = multierror.Append(failures, err)
        }
}
return failures.ErrorOrNil()
```